.evergreen/package-and-upload-artifact.sh

```diff
@@ -37,6 +40,10 @@ else
     # Verify signing
     spctl -a -vvv -t install dist/mongosh
   fi
+  if [ "$(uname)" == CYGWIN ]; then
+    # Fix absolute path before handing over to node
+    export ARTIFACT_URL_FILE=$(cygwin -w "$ARTIFACT_URL_FILE")
+  fi
```

Comment:
```suggestion
  if [ "$OS" == "Windows_NT" ]; then
    # Fix absolute path before handing over to node
    export ARTIFACT_URL_FILE="$(cygpath -w "$ARTIFACT_URL_FILE")"
  fi
```

(We use the same check for cygwin elsewhere too, and there might be spaces in the paths as well, even though that’s currently not a problem on evergreen itself I think)